splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/InsertOperationIT.java

```diff
@@ -466,13 +470,15 @@ public void testInsertFromSubOperation() throws Exception {
         Assert.assertEquals("Incorrect number of groups returned!", nameCountMap.size(), groupCount);
     }
 
+    @SuppressFBWarnings("RV_RETURN_VALUE_IGNORED_BAD_PRACTICE") // delete/createNewFile
     @Test
     public void testInsertBlob() throws Exception {
-        InputStream fin = new FileInputStream(getResourceDirectory() + "order_line_500K.csv");
-        PreparedStatement ps = methodWatcher.prepareStatement("insert into FILES (name, doc) values (?,?)");
-        ps.setString(1, "csv_file");
-        ps.setBinaryStream(2, fin);
-        ps.execute();
+        try(InputStream fin = new FileInputStream(getResourceDirectory() + "order_line_500K.csv")) {
+            PreparedStatement ps = methodWatcher.prepareStatement("insert into FILES (name, doc) values (?,?)");
```

Comment:
this ps and below rs should probably be closed, or surrounded by a try-with-resources, but it's unrelated to your commit

Reply:
methodWatcher.prepareStatement is keeping a list of statements and will close them automatically. Same btw for methodWatcher.execute and methodWatcher.executeQuery, making it much easier to make it SpotBugs-compliant.